routes/index.js

```diff
@@ -1,9 +1,57 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
+const util = require("util");
+
+// Database connection
+let knex = require("knex")({
+    client: "pg",
+    connection: {
+        host: process.env.DB_HOST,
+        port: process.env.DB_PORT,
+        user: process.env.DB_USER,
+        password: process.env.DB_PASS,
+        database: process.env.DB_NAME
+    }
+});
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get("/", function(req, res, next) {
+    res.render("index", { title: "Express" });
+});
+
+/* Example router to demonstrate some simple queries with knex */
+router.get("/examples/knex", function(req, res, next) {
```

Comment:
Because this endpoint actually `.send()`s, its best to not include the `next` parameter so that the `.send()` isnt executed multiple times